### PR TITLE
SLLS-190 Analyze ARM templates and Bicep files

### PR DESCRIPTION
**Pull request by jblievremont**

### CODE REVIEW

File: src/main/java/org/sonarsource/sonarlint/ls/AnalysisClientInputFile.java [hunk 1]
```diff
@@ -107,6 +107,7 @@ private static Language toSqLanguage(@Nullable String clientLanguageId) {
         // See https://github.com/forcedotcom/salesforcedx-vscode/blob/5e4b7715d1cb3d1ee2780780ed63f70f58e93b20/packages/salesforcedx-vscode-apex/package.json#L273
         Language.APEX;
       case "yaml" -> Language.YAML;
+      case "json" -> Language.JSON;
```
Comment (sophio-japharidze-sonarsource) [inline src/main/java/org/sonarsource/sonarlint/ls/AnalysisClientInputFile.java:110]: Why is this needed?

**Review by sophio-japharidze-sonarsource — APPROVED** [review]

File: src/main/java/org/sonarsource/sonarlint/ls/AnalysisClientInputFile.java (hunk 1 above)
Comment (jblievremont) [reply src/main/java/org/sonarsource/sonarlint/ls/AnalysisClientInputFile.java]: I assumed that it should be "forced" like for YAML, but apparently, both this line and the line above are not necessary. Good catch!
